app/controllers/cloud_network_controller.rb

```diff
@@ -65,6 +65,18 @@ def cloud_network_form_fields
     }
   end
 
+  def cloud_tenants_by_ems
```

Comment:
This PR adds 4 routines named `cloud_tenants_by_ems()` which have virtually identical body, the only difference
is in the `assert_privileges()` check.

Two things:
1. We want to avoid duplicates as much as possible
2. This cloud tenant info should be done via our rest api. @abellotti What do you think? Is this api already available?